recipes/libmysqlclient/all/test_v1_package/test_package.c

```diff
@@ -0,0 +1,8 @@
+#include <mysql.h>
```

Comment:
code duplication can be avoided. You can have a look to recipe templates in conan-center doc.